### ISSUE TITLE
iOS Bootcamp - Project 1 - Review  my app

### ISSUE THREAD

**Issue by kd3030** [kd3030/dropBoxWelcome#1]
@codepathreview

Please app is complete, please review /cc @codepathreview. 
This is for iOS for Designers and PM prework.
Week 1 Project: Dropbox

**Comment by chieger** [comment]
Nice work Kunal! 👍 

Try to challenge yourself to implement some of the optional features on the next assignment 😄 

~ Charlie
CodePath

**Comment by kd3030** [comment]
Charlie,

I tried to do the swipe as I was very much interested but was not able to
get it.  May be if you can please provide me some pointers during the call.

Thanks,
Kunal

**Comment by chieger** [comment]
Of course, I will be very happy to help you out during the Lab this Thursday! 😄 

Can you help remind me during the Thursday lab so I can make it a priority?

Scroll Views, Freeform View Controllers and Page Control can be a little daunting at first, but once you get the hang of it its a synch! 💪 

Best,
Charlie
